index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" 
+    content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles/index.css" >
+
+    <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css?family=Roboto|Crete Round|Poppins">
+    <title>Portfolio</title>
+</head>
+<body>
+  <header id="header">
+    <a>My Logo</a>  
```

Comment:
- [ ] As suggested by the previous reviewer that by convention this logo should be a link to the homepage. Now you have tried but there is something wrong with this syntax. kindly consider replacing line 16 with this code:
```
<a href="#">My Logo</a>
```